### PR TITLE
Add documentation for nav sim hotkeys

### LINKED ISSUE #431: Create a documentation page detailing all of the hotkeys
Include hotkeys from:
- [x] `Hotkeys.vue`
- [x] `DebugTools.vue`
- [x] `DrawModule.vue`
- [x] `FieldItems.vue`
- [x] Link to this page from the `simulators/nav` `README.md` page.

For each hotkey include
- [x] hotkey key code
- [x] description
- [x] file where hotkey is implemented

### PR DESCRIPTION
**Pull request by amverni**
resolves #431 

Add a wiki detailing what each hotkey does as well as what file that hotkey is implemented.

This changes was tested by viewing the wiki on my fork to verify it rendered properly.

### CODE REVIEW

**Review by raytitan — CHANGES_REQUESTED** [review]
Should these tiny commits be squashed together?

**Comment by amverni** [comment]
> Should these tiny commits be squashed together?

No. I will squash at the end. Up until then it won't matter. We can discuss this more offline if you'd like. It will also be laid out in the new workflow doc which I am concurrently working on. This change is for the example there.

**Comment by aswarner** [comment]
It looks like you didn't include hotkeys from FieldItems.vue, which #431 requests.

File: simulators/nav/src/components/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # <img src="../static/mrover.png" alt="MRover Logo" width="30"/> MRover Navigation Simulator
 
 ## Table of Contents
-[Directory Overview](#directory-overview)
```
Comment (aswarner) [inline simulators/nav/src/components/README.md:4]: lines 4-6 seemingly mistakenly removed a line break character, so these three headers render on one line in the wiki

File: simulators/nav/src/components/hotkeys.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+|| <div align="center">_LCM Messages_</div> ||
+| `shift+enter` | Turn the rover on/off. | `HotKeys.vue` |
+|| <div align="center">_Debug Tools_</div> ||
+| `shift+space` | Play/pause the simulator. | `HotKeys.vue` |
+| `shift+alt+space` | When the rover is paused, take a "step" by applying a single joystick command then pausing again. | `HotKeys.vue` |
+| `shift+up` | Apply a single drive forward joystick command. | `HotKeys.vue` |
+| `shift+down` | Apply a single drive backward joystick command. | `HotKeys.vue` |
+| `shift+left` | Apply a single turn left joystick command. | `HotKeys.vue` |
+| `shift+right` | Apply a single turn right joystick command. | `HotKeys.vue` |
+| `shift+r` | Turn off the rover and move it to the starting location. | `HotKeys.vue` |
```
Comment (aswarner) [inline simulators/nav/src/components/hotkeys.md:14]: this shift+r command is implemented in DebugTools.vue, not HotKeys.vue

**Review by aswarner — CHANGES_REQUESTED** [review]
I left two code comments that need revision. Additionally, this pull request does not address the fourth action item in #431 , which is to include the hotkeys from FieldItems.vue

File: simulators/nav/src/components/README.md (hunk 1 above)
Comment (amverni) [reply simulators/nav/src/components/README.md:4]: Good catch! My text editor must have removed this trailing whitespace without me realizing. I also noticed this problem in another nav sim readme so I'll fix both.

File: simulators/nav/src/components/hotkeys.md (hunk 2 above)
Comment (amverni) [reply simulators/nav/src/components/hotkeys.md:14]: Good catch on this one too!

**Comment by amverni** [comment]
> I left two code comments that need revision. Additionally, this pull request does not address the fourth action item in #431 , which is to include the hotkeys from FieldItems.vue

You're right! That should definitely be a part of this PR.

**Review by aswarner — APPROVED** [review]
Thanks for addressing my comments! Looks good to squash and merge

**Review by raytitan — APPROVED** [review]

**Review by DavidSmith166 — APPROVED** [review]

**Pull request merged**